marketplace/accounts/backends.py

```diff
@@ -1,7 +1,21 @@
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+from marketplace.accounts.models import User
```

Comment:
It is not interesting to import the user directly, this increases the coupling because if the user model is changed to another path, the functionality will no longer work.

An example of how this can be done: https://github.com/weni-ai/weni-integrations-engine/blob/430dbb555554b7ac4820e5a5611792e70b839f98/marketplace/core/serializers.py#L5

Reference: https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#referencing-the-user-model

Reply:
Done